test/olm/Dockerfile.registry

```diff
@@ -1,7 +1,9 @@
 FROM quay.io/openshift/origin-operator-registry:latest
 
-# The operator image location should be updated before copying.
+ARG image
```

Comment:
What about having a default?

Reply:
This `Dockerfile` is called from a `Makefile` target.  There is a default value set in that `Makefile`. The default value is `quay.io/redhat-developers/devopsconsole-operator`.

Reply:
Where exactly is it being passed to `docker build` in the Makefile?

Reply:
The code still not merged. It is here: https://github.com/redhat-developer/devconsole-operator/pull/33/files#diff-57dae1b5b313e109d7ebdfd77a500c5cR154

Reply:
But why did you then merge this PR before the other one?

Reply:
It was possible to create the image manually. I have described the steps in the original PR description. The manual steps are given here: https://github.com/redhat-developer/devconsole-operator/tree/master/test#steps-to-verify-operator-registry